rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
Yeah, this is not ideal. I meant to find some "high level" place where we would spec out the params once and only reference them like this in the `@ApiOperation` docs. I obviously forgot to do that ;)